docs/source/workflows/design_workflows.rst

```diff
@@ -74,6 +77,58 @@ Materials at the head of the list are the best materials found from searching th
 A workflow can be run using the python SDK.
 Triggering a workflow returns a workflow execution object.
 A workflow execution has a status (in progress, succeeded, or failed) and results (once execution has succeeded).
+
+Candidate results are paginated and returned as `DesignCandidate <#design-candidate>`__ objects.
+
+.. code:: python
+
+    from citrine.informatics.objectives import ScalarMaxObjective
+    from citrine.informatics.scores import LIScore
+    from citrine.jobs.waiting import wait_while_executing
+
+
+    # create a score with the desired objectives and baselines
+    score = LIScore(
+        # create an objective to maximize shear modulus
+        # the descriptor key must match a descriptor in materials produced from teh design space
+        objectives=[ScalarMaxObjective(descriptor_key='Shear modulus')],
+        baselines=[150.0] # one for each objective
+    )
+
+    # trigger a design run using a previously registered and validated workflow
+    execution = workflow.design_executions.trigger(score)
+
+    # wait for execution to complete
+    wait_while_executing(execution, print_status_info=True, collection=workflow.design_executions)
+
+    # get the candidate generator
+    execution_results = execution.candidates()
+
+    # pull out the candidate with the highest shear modulus and its score
+    # (this should be the candidate at the head of the list since we used shear modulus to score and rank materials)
+    # Note that because execution_results is a generator, calling this multiple times will iterate through the generator, getting the next best candidate
+    best_candidate = execution_results.send(None)
+    print(best_candidate)
+    best_score = best_candidate.primary_score
+    print(best_score)
+
+    # Alternatively, you can iterate over the candidates generator, looking at each candidate
+    for candidate in execution.candidates():
+        print(candidate.primary_score)
+
+    # To save all candidates in memory in one list:
+    all_candidates = list(execution.candidates())
+
+    # we can confirm the best candidate is at the head of the list using
+    # this candidate will be the same as best_candidate above
+    candidate_with_max_shear_modulus = max(all_candidates, key=lambda candidate: candidate.material.values['Shear modulus'].mean)
+    print(candidate_with_max_shear_modulus)
+
+
+.. warning::
```

Comment:
Added this warning, didn't want to remove the code example all together yet, but moved it below the preferred flow